test/jdk/java/foreign/libStdLibTest.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+#ifdef _WIN64
+#define EXPORT __declspec(dllexport)
+#else
+#define EXPORT
+#endif
+
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+EXPORT char *libc_strcat(char *str1, const char *str2);
+EXPORT int libc_strcmp(const char *str1, const char *str2);
+EXPORT size_t libc_strlen(const char *str);
+EXPORT int libc_puts(const char *str);
+EXPORT struct tm *libc_gmtime(const time_t *timer);
+EXPORT void libc_qsort(void *base, size_t nitems, size_t size, int (*compar)(const void *, const void*));
+EXPORT int libc_rand(void);
+EXPORT int vprintf(const char *format, va_list arg);
```

Comment:
This declaration of `vprintf` gives an error on Windows, is it needed? The test passes without as well.  (should came of stdio.h I'd think)

```
test\jdk\java\foreign\libStdLibTest.h(43): error C2375: 'vprintf': redefinition; different linkage
c:\progra~2\wi3cf2~1\10\include\100183~1.0\ucrt\stdio.h(738): note: see declaration of 'vprintf'
   ... (rest of output omitted)
```